src/components/Assets/AssetsTabComponent.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tab } from 'semantic-ui-react';
+import AssetsTableContent from '../AssetsTableContent';
+import FilterButton from '../common/FilterButton';
+import FilterComponent from '../common/FilterComponent';
+import PaginationComponent from '../common/PaginationComponent';
+import UploadAssetsContainer from '../../_components/Assets/UploadAssetsContainer';
+
+const AssetsTabComponent = (props) => {
+  const panes = [
+    {
+      menuItem: 'All Assets',
+      render: () => (
+        <Tab.Pane>
+          <FilterButton
```

Comment:
Hmm...this still seems longer.  https://github.com/AndelaOSP/art-dashboard/pull/174 have created a component called `UserFilterButton`, which have it's own container known as `UserFilterContainer`. Create a similar component in `Assets/` known  as `AssetsFilter` and it should also have it's own container. Then use the container here. This way, you will not be passing so many props down from parent to child, when a component can get the props from store. It is possible to use container components within presentational components. If in doubt, please ask me for clarifications tomorrow.